src/repl.rs

```diff
@@ -34,8 +34,8 @@ pub const FAILURE_MARKER: &str = "[tx:failed]";
 #[derive(Debug, Clone, Copy, PartialEq, Eq, clap::ValueEnum)]
 #[value(rename_all = "lowercase")]
 pub enum InputLanguage {
-    EdgeQL,
-    SQL,
+    EdgeQl,
+    Sql,
 }
```

Comment:
Sigh. Do linters feel really strongly about this one? I think this is an anti-improvement

Reply:
No, not strongly. We could of course disable this lint. It was mostly automatic. And I like that all things are cased the same.